src/common/commands_parser.rs

```diff
@@ -126,6 +129,7 @@ impl BuildEnclavesArgs {
         };
 
         Ok(BuildEnclavesArgs {
+            resources_dir: Path::new("/usr/share/nitro_enclaves/blobs/").to_path_buf(),
```

Comment:
Shouldn't this be configurable via `args`?

Reply:
Okay, I see this is only configurable when using the library, not when using the cli tool.